x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/prebuilt_rules/use_perform_rule_install.ts

```diff
@@ -44,18 +44,18 @@ const getSuccessToastMessage = (result: {
     failed: number;
   };
 }) => {
-  let toastMessage: string = '';
+  const toastMessages: string[] = [];
   const {
     summary: { succeeded, skipped, failed },
   } = result;
   if (succeeded > 0) {
-    toastMessage += i18n.INSTALL_RULE_SUCCESS(succeeded);
+    toastMessages.push(i18n.INSTALL_RULE_SUCCESS(succeeded));
   }
   if (skipped > 0) {
-    toastMessage += i18n.INSTALL_RULE_SKIPPED(skipped);
+    toastMessages.push(i18n.INSTALL_RULE_SKIPPED(skipped));
   }
   if (failed > 0) {
-    toastMessage += i18n.INSTALL_RULE_FAILED(failed);
+    toastMessages.push(i18n.INSTALL_RULE_FAILED(failed));
   }
-  return toastMessage;
+  return toastMessages.join(' ');
```

Comment:
nit: Little curious about the toast UX here since `skipped` + `failed` are included in the `onSuccess` toast (so long as the explicit `onError` hasn't occurred.)

Splitting out into separate toasts feels like toast spam, and doesn't look like we can add `\n`'s to toast titles, so it ends up looking something like this:

<img width="342" alt="image" src="https://github.com/elastic/kibana/assets/2946766/ad5580ff-d9c0-4661-b952-8637991e7aaf">

Maybe a toast w/ title + description would work here if `toastMessages.length > 1`, and then have each message on a new line in the description? Could also color a `warning` toast if `skipped` or `failed` were > 0?